packages/core/src/util/calcStabilityWindow.ts

```diff
@@ -0,0 +1,9 @@
+// 3k/f (where k is the security parameter in genesis, and f is the active slot co-efficient parameter
```

Comment:
This should be a
```
/**
 * doc comment on calculateStabilityWindowSlotsCount
 */
```

Reply:
c66f5797594c56d155aeb62836e4228622916a55